Test.java

```diff
@@ -0,0 +1,12 @@
+class MyClass {
+  MyClass(MyClass mc) { }
+  
+  int     foo1() { return 0; }
+  void    foo2(int value) { }
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)